.travis.yml

```diff
@@ -7,76 +7,10 @@ sudo: required
 
 matrix:
     include:
-        - dist: trusty
-          compiler: gcc
-          env: wxGTK_VERSION=2 wxUSE_XVFB=1
-          name: wxGTK 2 Ubuntu 14.04
-        - dist: xenial
-          compiler: gcc
-          env: wxGTK_VERSION=3
-          name: wxGTK 3 Ubuntu 16.04
-        - dist: bionic
-          compiler: gcc
-          env: wxGTK_VERSION=2 wxCONFIGURE_FLAGS="--enable-utf8 --enable-utf8only --enable-monolithic" wxUSE_XVFB=1
-          name: wxGTK 2 UTF-8 Ubuntu 18.04
-        - dist: bionic
-          compiler: gcc
-          env: wxGTK_VERSION=3 wxCONFIGURE_FLAGS="--enable-cxx11 --enable-stl --disable-webview" wxMAKEFILE_FLAGS="CXXFLAGS=-std=c++11 LDFLAGS=-Wl,--no-as-needed" wxUSE_XVFB=1
-          name: wxGTK 3 STL Ubuntu 18.04
-        - dist: bionic
-          compiler: clang
-          env: wxGTK_VERSION=2 wxCONFIGURE_FLAGS="--disable-shared --disable-sys-libs --disable-webview" wxUSE_XVFB=1
-          name: wxGTK 2 clang Ubuntu 18.04
-        - dist: bionic
-          compiler: gcc
-          env: wxGTK_VERSION=3 wxTOOLSET=cmake wxCMAKE_GENERATOR="Unix Makefiles"
-          name: wxGTK 3 CMake Ubuntu 18.04
-        - os: osx
-          osx_image: xcode6.4
-          compiler: clang
-          env: wxCONFIGURE_FLAGS="--enable-cxx11" wxMAKEFILE_FLAGS="CXXFLAGS=-std=c++11" wxSKIP_SAMPLES=1
-          name: wxOSX Xcode 6.4
-        - os: osx
-          osx_image: xcode9.4
-          compiler: clang
-          env: wxTOOLSET=cmake wxCMAKE_GENERATOR=Xcode wxCMAKE_DEFINES="-DCMAKE_CXX_STANDARD=11"
-          name: wxOSX CMake Xcode 9.4
-        - os: osx
-          osx_image: xcode11.3
-          env: wxCONFIGURE_FLAGS="--enable-monolithic --with-cxx=17 --with-macosx-version-min=10.12 --enable-stl"
-          name: wxOSX Xcode 11.3
         - os: osx
           osx_image: xcode11.4
-          env: wxCONFIGURE_FLAGS="--enable-monolithic --with-cxx=17 --host=i686-apple-darwin_sim --build=x86_64-apple-darwin17.7.0 --with-osx_iphone --with-macosx-version-min=9.0 --with-macosx-sdk=$(xcrun --sdk iphonesimulator --show-sdk-path) --enable-stl --with-libtiff=builtin" wxSKIP_GUI=1 wxSKIP_SAMPLES=1
+          env: wxCONFIGURE_FLAGS="--enable-monolithic --with-cxx=17 --host=i686-apple-darwin_sim --build=x86_64-apple-darwin17.7.0 --with-osx_iphone --with-macosx-version-min=10.0 --with-macosx-sdk=$(xcrun --sdk iphonesimulator --show-sdk-path) --enable-stl --with-libtiff=builtin" wxSKIP_GUI=1 wxSKIP_TESTING=1 wxSKIP_SAMPLES=1
```

Comment:
Curious, why do we need `--with-libtiff=builtin`? I.e. why do we need it for this one and why don't we need it for the other 3rd party libs?

Reply:
I have no idea, but the build broke, it somehow found a system installed tiff include dir, but then failed include tiff.h, the other picture libs were built-in, and also when building on my system configure configure uses the built-in automatically

Reply:
configure apparently thinks `-I/usr/local/Cellar/libtiff/4.1.0/include` is perfect, but then it fails with` ./src/common/imagtiff.cpp:40:14: fatal error: 'tiff.h' file not found`

[build log](https://travis-ci.org/github/wxWidgets/wxWidgets/jobs/684169612)

Reply:
I see, thanks. I need to look into the proper use of `pkg-config` when cross-compiling, but for now using this option or, better, `--disable-sys-libs` (because system libz seems to be also found), is indeed the right thing to do.

Reply:
the libtiff is the only header that is picked up from the outside, I have no idea why. Actually zlib is correctly taken from `-isysroot`

`/Applications/Xcode-11.4.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/usr/include/zlib.h`

but then again, there is a problem - and we get a proper warning - because according to our deployment settings, inflateValidate may not be available, since the guards are evaluated using macros from the zlib from the SDK, not from the lowest version. So I'll switch that to `-disable-sys-libs`, thanks